src/server/routes/verifyEmail.ts

```diff
@@ -183,6 +183,9 @@ router.get(
           addQueryParamsToPath(
             `${consentPages[0].path}`,
             res.locals.queryParams,
+            {
+              useIdapi: true,
```

Comment:
It's because the `/verify-email` routes are only IDAPI specific, and only set the IDAPI cookies when a user verifies their email using this flow, so no Okta session. So when redirecting to the onboarding flow from the `/verify-email` routes we have to make sure that `useIdapi` param is set so that it uses the makes sure to use IDAPI logic to check if the user is logged in, and not the Okta session.